lxd/instance_post.go

```diff
@@ -73,6 +73,12 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 				return errors.Wrap(err, "Failed to load LXD config")
 			}
 
+			// Check if user is allowed to use '--target' flag
```

Comment:
Similar to the error message, I'd just put `Check if user is allowed to use cluster member targeting`